pkg/operator/operator.go

```diff
@@ -274,6 +275,13 @@ func (c *vSphereProblemDetectorController) runSingleNodeSingleCheck(checkContext
 	resultCollector.AddResult(res)
 }
 
+func (c *vSphereProblemDetectorController) finishNodeChecks() {
```

Comment:
Can we add a comment in `FinishCheck` function that - as a general rule one should not perform expensive operator in `FinishCheck` because they are run in serial and can block operator for long time?

Reply:
added  comment + full Context to be able to interrupt any computation.